static/lib/tutorials/en_US/expresstohapi.md

```diff
@@ -174,7 +174,7 @@ hapi has the functionality to respond with JSON data by default. The only thing
 
 ## <a name="plugins" ></a> Middleware vs Plugins and Extensions
 
-To extend its functionality, Express uses middleware. Middleware essentially is a sequence of functions using callbacks to execute the next function. The issue with this is as your application grows in size and complexity, the order at which middleware executes becomes more crucial and more difficult to maintain. Having a middleware execute before one it's dependant on will cause you application to fail. hapi fixes this issue with its robust plugin and extension system. 
+To extend its functionality, Express uses middleware. Middleware essentially is a sequence of functions using callbacks to execute the next function. The issue with this is as your application grows in size and complexity, the order at which middleware executes becomes more crucial and more difficult to maintain. Having a middleware execute before one it is dependant on will cause your application to fail. hapi fixes this issue with its robust plugin and extension system. 
```

Comment:
Thank you for the fixes. `it's` is actually a contraction of `it is` so that's not an error here but I agree for the `your`. Anyway it's more readable that way.